config/certmanager/certificate.yaml

```diff
@@ -6,15 +6,15 @@ apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
   name: selfsigned-issuer
-  namespace: system
+  namespace: openshift-sandboxed-containers-operator
```

Comment:
What's the reason for that ? This comes from Operator SDK AFAIK. Not sure it should be changed...